BigCommerce/UserInterface/01a_Connections.md

```diff
@@ -0,0 +1,198 @@
+# CONNECTIONS (BigCommerce)
+
+The CONNECTIONS tab offers an overview of all connections created for *Omni-Channel*. It also allows configuring these connections. 
+
+*Omni-Channel > Settings > Tab CONNECTIONS*
+
+![Connections](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/Connections.png "[Connections]")
+
+**Connections**
+
+The *CONNECTIONS* tab contains a list view of all available connections.
+The following functions are available in the list header of the *CONNECTIONS* tab:
+
+- ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh)   
+    Click this button to update the list of connections.
+
+- *VIEW*   
+    Click the drop-down list to select the view. All created views are displayed in the drop-down list. For a detailed description of how to create and edit a list view, see [Work with lists](../../Core1Platform/UsingCore1/04_WorkWithLists.md).  <!---Preliminary link-->
+
```

Comment:
Beschreibung zu Columns-Button fehlt hier:

- ![Columns](../../Assets/Icons/Columns.png "[Columns]") Columns (x)   
    Click this button to display the columns bar and customize the displayed columns and the order of columns in the list. The *x* indicates the number of columns that are currently displayed in the list.

Reply:
done